source

```diff
@@ -102632,18 +102636,21 @@ dictionary <dfn>StorageEventInit</dfn> : <span>EventInit</span> {
    <dd>Switch to the <span>data state</span>. Emit the current tag token.</dd>
 
    <dt>U+0000 NULL</dt>
-   <dd><span>Parse error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current
+   <dd><span data-x="parse-error-unexpected-null-character">unexpected-null-character</span><span>parse
+   error</span>. Append a U+FFFD REPLACEMENT CHARACTER character to the current
    attribute's value.</dd>
 
    <dt>U+0022 QUOTATION MARK (&quot;)</dt>
    <dt>U+0027 APOSTROPHE (')</dt>
    <dt>U+003C LESS-THAN SIGN (&lt;)</dt>
    <dt>U+003D EQUALS SIGN (=)</dt>
    <dt>U+0060 GRAVE ACCENT (`)</dt>
-   <dd><span>Parse error</span>. Treat it as per the "anything else" entry below.</dd>
+   <dd>This is an <span data-x="parse-error-unexpected-character-in-attribute-value">unexpected-character-in-attribute-value</span>
```

Comment:
We need to be more specific here: these character are unsafe only in unquoted attribute values and could be a sign of missing quotes or malformed markup, so it better be: `unexpected-character-in-unquoted-attribute-value`